alicloud/resource_alicloud_db_instance.go

```diff
@@ -254,7 +254,7 @@ func resourceAlicloudDBInstanceCreate(d *schema.ResourceData, meta interface{})
 	d.SetId(response.DBInstanceId)
 
 	// wait instance status change from Creating to running
-	stateConf := BuildStateConf([]string{"Creating"}, []string{"Running"}, d.Timeout(schema.TimeoutCreate), 5*time.Minute, rdsService.RdsDBInstanceStateRefreshFunc(d.Id(), []string{"Deleting"}))
+	stateConf := BuildStateConf([]string{"Creating"}, []string{"Running"}, d.Timeout(schema.TimeoutCreate), 30*time.Second, rdsService.RdsDBInstanceStateRefreshFunc(d.Id(), []string{"Deleting"}))
```

Comment:
I think the 30 seconds is too short. A new rds instance will cost at least 3 minutes. I think it can be change to 3*time.Minute.

Reply:
@xiaozhu36 
This is the polling interval. Shouldn’t it be necessary to wait manually for 3 minutes before polling?

Reply:
Yes, creating one instance will cost at least 3 min and there is no need to poll in the known pending stage. Too many polling will cost the API and user's throttling threshold.

Reply:
@xiaozhu36 
already edited